### ISSUE TITLE
Support Ordinals on testnet and signet

### ISSUE THREAD

**Issue by 314159265359879** [blockstack/blockstack-app#4551]
The ordinalsbot team reached out we could use this API

_Right now you are redirecting to generative.xyz mainnet and it's not working, ours will work for test net:
https://testnet-explorer.ordinalsbot.com/tx/485d3b042a33b3457fadb0de323bfdc2fac86ad01f86752e90b447743fa12c6c_

[Original conversation on Discord](https://discord.com/channels/1075856098269089834/1084803828618113055/1171098880868352041)

Hiro Systems has a meeting this week to see if they can prioritize adding an API-support for testnet ordinals https://github.com/hirosystems/bitcoin-indexer/issues/495

**Comment by markmhendrickson** [comment]
> Right now you are redirecting to generative.xyz mainnet and it's not working

Should we label this as a bug if current functionality is broken?

**Comment by 314159265359879** [comment]
@markmhendrickson yes I will classify it as a P2 bug, seems like there is quite a few developers looking forward to an update that allows showing testnet ordinals. 

I see no updates in the issue for the enhancement of the ordinals api from Hiro so this Ordinalsbot API for testnet seems like a good option to role out a fix quickly?

**Comment by markmhendrickson** [comment]
I'm moving to the dev backlog as an urgent bug to resolve

**Comment by edgarkhanzadian** [comment]
Maybe we can try gamma's ordinals testnet: https://ordinals-testnet.gamma.io/

**Comment by donut33-social** [comment]
Is there any new update?

**Comment by markmhendrickson** [comment]
@kyranjamie would this require an UI work to support, or can we simply hardwire Gamma's testnet URL above if the user is in testnet mode?

**Comment by kyranjamie** [comment]
We query a user's inscriptions from @hirosystems API, so for an "easy" implementation we'd need a testnet version of their API.

If we have any other testnet API we can use—with some more involved mapping to the same schema—we can add testnet support. Gamma's instance of Ordinals.com on testnet isn't much help without an API.

Edit: note BIS does have a testnet API

**Comment by markmhendrickson** [comment]
https://testnet.api.bestinslot.xyz and https://signet_api.bestinslot.xyz
